engine/gui/src/test/test_gui.cpp

```diff
@@ -2813,8 +2813,8 @@ TEST_F(dmGuiTest, DeltaTime)
 
 TEST_F(dmGuiTest, Bug352)
 {
-    dmGui::AddFont(m_Scene, "big_score", 0, 0);
-    dmGui::AddFont(m_Scene, "score", 0, 0);
+    dmGui::AddFont(m_Scene, dmHashString64("big_score"), 0, 0);
+    dmGui::AddFont(m_Scene, dmHashString64("score"), 0, 0);
```

Comment:
I want a lua test as well for the go.set() with keys

Reply:
yes, it's important for sure. I'll do